Eevee.Sleep.Bot/Handlers/EventHandlers/GuildMemberUpdatedEventHandler.cs

```diff
@@ -135,12 +135,19 @@ SocketGuildUser updated
             .Where(x => rolesAdded.Contains(x.RoleId))
             .ToHashSet();
 
+        if (rolesAdded.Length > 0)
+        {
+            await UserRoleContoller.AddRoles(updated.Id.ToString(), rolesAdded);
+        }
+
         if (taggedRolesAdded.Count > 0) {
             await HandleRolesAdded(client, environment, taggedRolesAdded, updated);
             return;
         }
 
         if (rolesRemoved.Any(taggedRoles.Select(x => x.RoleId).Contains)) {
+            // I don't know it needs to be removed from the database
```

Comment:
No need as we're effectively "role record" which records if an user ever obtained a role before.